src/zx/FunctionalityConstruction.cpp

```diff
@@ -223,6 +226,65 @@ FunctionalityConstruction::parseOp(ZXDiagram& diag, op_it it, op_it end,
       addZSpider(diag, target2, qubits, PiExpression(), EdgeType::Hadamard);
       break;
     }
+    case qc::OpType::RZZ: {
+      const auto ctrl = target;
```

Comment:
Now that there is no CNOT call here any more there is also no need for this copy any more (not that it was necessary from the beginning).

Reply:
It was necessary because otherwise the linter complains that I use a variable called `target` for a parameter called `control`.

Reply:
Just to clarify: clang-tidy is not authoritative in the sense that every warning it spits out has to be taken as gods word. It is totally fine to disagree with some of the warnings and ignore them on individual lines (as long as that does not become the norm and there is a reason for it).